File: backend/src/main/kotlin/bo/edu/umsa/backend/dto/TaskReportDto.kt
package bo.edu.umsa.backend.dto

import java.sql.Timestamp

data class TaskReportDto(
    val taskId: Int,
    val taskName: String,
    val projectName: String,
    val taskStatusName: String,
    val taskPriorityName: String,
    val taskCreationDate: Timestamp,
    val taskDueDate: Timestamp,
    val taskEndDate: Timestamp?,
    val taskRating: Int,
    val replacedPartDescription: String,
    val taskAssignees: List<String>,
)

File: frontend/src/app/features/report/models/task-report.dto.ts
export interface TaskReportDto {
    taskId: number;
    taskName: string;
    projectName: string;
    taskStatusName: string;
    taskPriorityName: string;
    taskCreationDate: Date;
    taskDueDate: Date;
    taskEndDate: Date;
    taskRating: number;
    replacedPartDescription: string;
    taskAssignees: string[];
}
